src/query/service/src/pipelines/processors/transforms/hash_join/hash_join_build_state.rs

```diff
@@ -101,6 +111,10 @@ pub struct HashJoinBuildState {
     pub(crate) send_val: AtomicU8,
     /// Wait all processors finish read spilled data, then go to new round build
     pub(crate) restore_barrier: Barrier,
+    pub(crate) enable_inlist_runtime_filter: bool,
+    pub(crate) enable_min_max_runtime_filter: bool,
+    /// Need to open runtime filter setting.
```

Comment:
for inlist and min_max index, we really need a flag to control them? I think they can always be open.

Reply:
Still have some limitations: https://github.com/datafuselabs/databend/pull/13993/files#diff-3a9124588945cc9f4c83fd0bfef409c1fc3ef74b2ccfd49e2504af36bbbb3a9dR144-R157